### DISCUSSION TITLE
Create Form

### DISCUSSION THREAD

**Discussion by Dr-Maspix** [filamentphp/filament#4109] [Help]
Dears , 

Kindly please help as i need create Resource With Form Only 
as per after i click link on sidebar i need to go to direct to insert or update form not table ,

Regards

**Comment by danharrin** [comment]
Create a custom page, and use the form builder. No resource.

https://filamentphp.com/docs/2.x/admin/pages/getting-started#creating-a-page
https://filamentphp.com/docs/2.x/forms/getting-started
